lib/Command/MigrateShare.php

```diff
@@ -95,5 +95,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$output->writeln("<error>{$e->getMessage()}</error>");
 			return self::ERROR_CONVERSION_EXCEPTION;
 		}
+
+		return 0;
```

Comment:
`execute()` is supposed to return an int. When we get to here the command has happened - better return 0 (success)